### PR TITLE
Sale on Hold banner 📱

### PR DESCRIPTION
**Pull request by ashfurrow**
Supersedes #2327.

This was a joint effort between myself and @erikdstock. There is a new sale-on-hold indicate that replaces the title of the auction when the operator places the sale on hold:

simulator screen shot jun 9 2017 2 44 08 pm
simulator screen shot jun 9 2017 2 43 23 pm

( @katarina getting the indicator centred on iPad was really difficult, let me know if the above is acceptable, or I can revisit)

When the sale first goes on hold _or_ when the user first opens the live interface, they're presented with an overlay with a brief explanation. They can return to this overlay by tapping the red "sale on hold" banner.

simulator screen shot jun 9 2017 2 41 02 pm
simulator screen shot jun 9 2017 2 43 19 pm

### CODE REVIEW

File: Artsy_Tests/View_Controller_Tests/SaleOnHoldOverlayViewTests.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                let containerView = UIView().then {
+                    $0.backgroundColor = .gray
+                    $0.frame = UIScreen.main.bounds
+                }
+
+                let subject = SaleOnHoldOverlayView()
+                containerView.addSubview(subject)
+                subject.align(toView: containerView)
+
+                expect(containerView).to( recordSnapshot() )
```
Comment (ashfurrow) [inline Artsy_Tests/View_Controller_Tests/SaleOnHoldOverlayViewTests.swift]: Oops, left these in by mistake.

File: Artsy_Tests/View_Controller_Tests/Live_Auction/LiveAuctionViewControllerTests.swift [hunk 2]
```diff
@@ -54,9 +54,11 @@ class LiveAuctionViewControllerTests: QuickSpec {
 
         it("handles splitting in an iPad") {
             setupViewControllerForPhone(false)
-            subject.view.frame = CGRect(x: 0, y: 0, width: 1024, height: 768)
+            ARTestContext.use(.pad) {
+                subject.view.frame = CGRect(x: 0, y: 0, width: 1024, height: 768)
 
-            expect(subject).to (haveValidSnapshot(named: nil, usesDrawRect: true))
+                expect(subject).to (haveValidSnapshot(usesDrawRect: true))
+            }
```
Comment (ashfurrow) [inline Artsy_Tests/View_Controller_Tests/Live_Auction/LiveAuctionViewControllerTests.swift]: This snapshot test was always wrong (see changes made to the reference image and in ARTestContext). It's been fixed now.

Comment (orta) [inline Artsy_Tests/View_Controller_Tests/Live_Auction/LiveAuctionViewControllerTests.swift]: 👍

File: Podfile.lock [hunk 3]
```diff
@@ -452,7 +452,7 @@ SPEC CHECKSUMS:
   Extraction: 612cf0866f74d4c0dd616677ff24146efa200958
   FBSDKCoreKit: d2aaed5e9ab7d8d6301c533376a1fbff1cf3deb5
   FBSDKLoginKit: 699ff169080e3072de4b9b0faca90bf23dc36deb
-  FBSnapshotTestCase: '094f9f314decbabe373b87cc339bea235a63e07a'
+  FBSnapshotTestCase: 094f9f314decbabe373b87cc339bea235a63e07a
```
Comment (orta) [inline Podfile.lock]: heh, all these strings

**Pull request merged**